docs/modules/ROOT/pages/reference-commands.adoc

```diff
@@ -0,0 +1,281 @@
+
+m| +:config {}+
+a| Reset the configuration settings to the default configuration.
+
+m| +:config {<key-value pairs>}+
+a|
+Reset the configuration settings to default and then set the key-value pairs. +
+The key-value pairs can be specified as `+{"x": 1, "y": 2}+`, `+{'x': 1, 'y': 2}+`, or `+{x: 1, y: 2}+`.
+
+m| :dbs
+a| Show databases available for the current user.
+
+m| +:delete <request>+
+a| REST command, send a HTTP DELETE request.
+
+m| :edit
+a| Create an edit frame.
+
+m| +:get <request>+
+a| REST command, send a HTTP GET request.
+
+m| :help
+a| Display help about topics and commands.
+
+m| +:head <request>+
+a| REST command, send a HTTP request with specified header.
+
+m| :history
+a| Display Neo4j Browser history of your executed commands.
+
+m| :history clear
+a| Remove all commands from Neo4j Browser history.
+
+m| +:put <request>+
+a| REST command, send a HTTP PUT request.
+
+m| :play
+a| Play tutorials and guides.
+
+m| +:post <request>+
+a| REST command, send a HTTP POST request.
+
+m| +:param <key-value>+
+a|
+Set one parameter. +
+The key-value can be specified as `+"x" => 1+` or `+x => 1+`.
+
+m| :params
+a| Show all parameters.
+
+m| +:params {}+
+a| Remove all parameters.
+
+m| +:params {<key-value pairs>}+
+a|
+Remove all parameters and then set the parameters to the specified key-value pairs. +
+The key-value pairs can be specified as `+{"x": 1, "y": 2}+`, `+{'x': 1, 'y': 2}+`, or `+{x: 1, y: 2}+`.
+
+[NOTE]
+====
+Integers will be set as floats in this form.
+====
+
+m| :queries
+a| List your servers and clusters that are running queries.
+
+m| :style
+a| Show the current style configuration.
+
+m| +:style <CSS>+
+a|
+Modify the visual aspects of your graph. +
+CSS classes will be overwritten with the new styling.
+
+m| :style reset
+a| Reset the style to the default styling.
+
+m| :server change-password
+a| Change password.
+
+m| :server connect
+a| Manage a connection to Neo4j.
+
+m| :server disconnect
+a| Disconnect the current open connection to Neo4j.
+
+m| :server status
+a| Show metadata for the current open connection.
+
+m| :schema
+a| Show information about database schema indexes and constraints.
+
+m| :sysinfo
+a| Show information about _Store Size_, _Id Allocation_, _Page Cache_, _Transactions_, and _Databases_.
+
+m| +:use <database>+
+a|
+Set the database, to query to by default (the default database is set to `neo4j`).
+
+[NOTE]
+====
+You can also use Cypher, `+USE <database> ...+`, to specify a database.
+====
+
+//m| :snake
+//a| Play snake. Easter egg.
+
+|===
+
+
+
+
+[[configuration-commands]]
+== Configuration commands
+
+To set a new value for a Neo4j Browser configuration use the syntax:
+[source, shell, role=noheader]
+----
+:config "<configurationName>": <configurationValue>`
+----
+
+
+.Set the `maxHistory` to `20`.
+====
+[source, shell, role=noplay]
+----
+:config "maxHistory": 20
+----
+====
+
+
+.Show the configuration settings.
```

Comment:
I've moved this together with example output to my PR, so you can remove the whole section Configuration commands from here.